README.md

```diff
@@ -107,25 +107,126 @@ Each of these can be run on an individual well using a plain `bash` script or as
 
+> [!NOTE]
+> The paths can be *relative* to the scripts, as is in the example provided here, which assumes the layout of the repository is fixed. Otherwise, the paths should be provided as **absolute paths**.
+
+## Usage
+
+We have made an example dataset with one well data publicly available. The well row number=3 and column number=7. The image has 3 channels, channel1=EGFP, channel2=mCherry and channel3=brightfield. 
+
+It can be downloaded from: https://figshare.com/articles/dataset/cellos_data_zip/21992234  
+
+On Linux, you can download it as follows:
+
+```bash
+wget https://figshare.com/ndownloader/files/39032216
+```
+
+> [!WARNING]
+> This is a ~11Gb zip file. 
+
+On Linux, it needs to be unziped using [7z](https://www.7-zip.org):
 
-> *Note*: you have to edit the config file to your needs. 
->
-> | Parameter | Description | 
-> |-----|-------------|
-> | plate_path   | path to where your raw images are | 
-> | output_path   | path to where the csv files and zarr arrays will be saved   | 
-> | well_targets   | name number of rows and columns (row1,col1&#124;row2,col2) of wells to analyze   | 
-> | plane_size   |  size of image of one field, one z-slice and one channel   | 
-> | overlap_x and y   | overlapping pixels between two adjacent fields   | 
-> | stardist_path   | path to the trained model for nuclei segmentation  | 
-> | ...  | ...  | 
+```bash
+7z x 39032216
+```
+
+This will extract a `cellos_data` folder consisting of images (`.tiff`) and `Index.idx.xml` (metadata) file.
+
+To use the provided `config.example.cfg` and script commands from above, we recommend you place the `cellos_data` in the root of this repository.
+
+You should obtain the following layout for the `Cellos` directory, where `...` indicates abridged files:
 
+```
+├── config.example.cfg
+├ ...
+├── cellos_data
+│   └── Index.idx.xml
+|   └── r03c07 ... .tiff
+├── models
+│   └── stardist
+│       ├ ...
+├── output
+│   ├ ...
+└── scripts
+    ├── process_cells
+    │   ├── cells_process_plate.sh
+    │   ├── cells_seg_well.py
+    │   └── cells_seg_well.sh
+    └── process_organoids
+        ├── process_plate.sh
+        ├── stitch_well.py
+        └── stitch_well.sh
+```
 
-> ***Tip***: All of the steps implemented in our pipeline are optimized to run on high-performance computer (HPC) systems to take advantage of parallel processing and to carry out the steps that are computationally intensive. The most expensive step in the pipeline is to process the whole image in the steps to organize the image and segment organoids. To minimize this, if not needed, you can remove the step to calculate area of image that has organoids and to plot raw images. To run organoids segmentation step on demo data we used a wall-clock time of approximately 2 hours and 30 min on an HPC system using 1 core and 32 GB of memory and to run nuclei segmentation step we used a wall-clock time of approximately 25 min on HPC system using 4 cores and less than 5 GB of memory. 
```

Comment:
I deleted this because I now provide info about timing and memory in the individual steps. Also Im not sure the pipeline is so optimized per se! 🤣